types/office-js/index.d.ts

```diff
@@ -63507,6 +63671,20 @@ declare namespace Excel {
              * [Api set: ExcelApi 1.10]
              */
             id?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Gets the entities (e.g. people) that are mentioned in comments.
```

Comment:
```suggestion
             * For EACH ITEM in the collection: Gets the entities (e.g., people) that are mentioned in comments.
```

Also update other instances if you take this change

Reply:
I'll file a longer-term issue to fix that. It's present in every such method throughout the API. It's also hidden in our docs, so ¯\_(ツ)_/¯